docs/content/how-to-develop-a-workflow-definition-in-go.md

```diff
@@ -77,6 +77,8 @@ func YourWorkflowDefinition(ctx workflow.Context, param YourWorkflowParam) (Your
 ```
 
 Returning a non-nil `error` from a Workflow indicates that an error was encountered during its execution and the Workflow Execution should be [Terminated](#).
+This will also cause any custom return value to be ignored.
+It is not possible for both a non nil error and a custom return value from a Workflow Execution to be received as a return values, from the Workflow Execution, as is possible in pure Go.
```

Comment:
```suggestion
It is not possible for both a non-nil error and a custom return value from a Workflow Execution to be received as return values from the Workflow Execution, as is possible in pure Go.
```
Or rewrite to avoid "It is":
A Workflow Execution cannot return both a non-nil error and a custom return value, something that is possible in pure Go.

Reply:
Ya, the tricky thing here is that it CAN return both from the Definition... its just that only one or the other will be available to the caller who is looking for the result.